config/peerpods/podvm/bootc/usr/lib/bootc/install/00-kargs.toml

```diff
@@ -0,0 +1,5 @@
+[install.filesystem.root]
+type = "ext4"
+[install]
+kargs = [ "console=ttyS0", "selinux=0", "enforcing=0", "audit=0"]
```

Comment:
We'll need to enable selinux eventually. May be worth testing without adding the following args `"selinux=0", "enforcing=0", "audit=0"` ?